### PR TITLE
Building with responsive design. Project #6 Microverse.

### PR DESCRIPTION
**Pull request by SigmaSam**

### CODE REVIEW

**Comment by SigmaSam** [comment]
### Replicated the TNW website.
- Build an HTML and CSS responsive design.
- Worked with Grid and Flex to create a responsive design.
- Gave the website a breakpoint at 76Xpx screen width to be adapted to smaller devices.

**Comment by SigmaSam** [comment]
### Replicated the TNW website.
- Build an HTML and CSS responsive design.
- Worked with Grid and Flex to create a responsive design.
- Gave the website a breakpoint at 76Xpx screen width to be adapted to smaller devices.

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @SigmaSam & @jacobo12montes  :wave:
-
You have done a splendid job! :tada: Keep it up.
We, however, do have some few changes to carry out before we proceed (see review comment).
##
## Required
- [x] In the top left main article, the date and the main red gradient are missing. Please add them. See the difference below 👇 

|[Source](https://thenextweb.com/)|[Clone](https://rawcdn.githack.com/SigmaSam/Building-with-Responsive-Design-Samuel-Juan/f49fa110b5115a7ca02139dd883880e176a1ab00/index.html)|
|:---:|:---:|
|large_screen|large_screen-clone|

### Medium Screen Changes
- [x] The contents should be fitting the entire width of the view. Currently, we have some unnecessary space (margin) in the following sections. See the changes below 👇 

- The latest news section:

|[Source](https://thenextweb.com/)|[Clone](https://rawcdn.githack.com/SigmaSam/Building-with-Responsive-Design-Samuel-Juan/f49fa110b5115a7ca02139dd883880e176a1ab00/index.html)|
|:---:|:---:|
|medium-screen|medium-screenclone|

- The two article section: 

|[Source](https://thenextweb.com/)|[Clone](https://rawcdn.githack.com/SigmaSam/Building-with-Responsive-Design-Samuel-Juan/f49fa110b5115a7ca02139dd883880e176a1ab00/index.html)|
|:---:|:---:|
|two-articles-medium|two-articles-medium-clone|

- There is a slight difference in the latest deals section

|[Source](https://thenextweb.com/)|[Clone](https://rawcdn.githack.com/SigmaSam/Building-with-Responsive-Design-Samuel-Juan/f49fa110b5115a7ca02139dd883880e176a1ab00/index.html)|
|:---:|:---:|
|latest-deals-medium|latest-deals-clone|
##
### Small Screen Changes
- [x] In the small screen changes, all images take the maximum width and should be aligned in one column ([link to page](https://thenextweb.com/)).
- [x] Navigation and sub-navigation row changes to a hamburger (bars) icon at the rightmost part (next to search icon) ([link to page](https://thenextweb.com/))

|[Source](https://thenextweb.com/)|[Clone](https://rawcdn.githack.com/SigmaSam/Building-with-Responsive-Design-Samuel-Juan/f49fa110b5115a7ca02139dd883880e176a1ab00/index.html)|
|:---:|:---:|
|small-screen|small-screen-clome|

- [x] Make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright).

- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

File: index.html [hunk 1]
```diff
@@ -0,0 +1,509 @@
...
+      <div class="face__articles">
+        <div class="face__one">
+          <div>
+            <img class="face__logo" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="">
+            <p class="face__logo2">Friday — May 22, 2020</p>
+          </div>
+          <p class="face__insigh">insights</p>
+          <a class="face__art" href="#">4 vital lessons I learned from my Series A fundraisng journey</a>
+          <p class="face__author">by Neha Sampat 32 minutes ago</p>
+        </div>
```
Comment (lucasmazo32) [inline index.html:65]: - [ ] The logo in the first article should not appear when the logo appears in the navbar.
> Source
 

>Clone

File: index.html [hunk 2]
```diff
@@ -0,0 +1,509 @@
...
+
+        <div class="face__three">
+          <p class="face__insigh3">APPS</p>
+          <a class="face__art3" href="#">How to transfer your Google Play Music library to YouTube Music</a>
+          <p class="face__author3">by Rachel Kaser 7 hours ago</p>
+        </div>
+      </div>
+
+      <div class="main__flex">
+        <h2 class="lastest__title"><a href="#">Lastest news ></a></h2>
```
Comment (lucasmazo32) [inline index.html:79]: - [ ] Kindly fix the layout of the Latest news section for medium screens.

> Source 
image


> Clone
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,509 @@
...
+                <div class="last__description">
+                  <p>
+                    Modulr raises £18.9M for its ‘Payments as a Service’ on TechCrunch
+                  </p>
+                </div>
+          </div>
+        </div>
+
+
+          <div class="categories">
```
Comment (lucasmazo32) [inline index.html:231]: - [ ] Kindly fix the layout for medium screens:

> Source 
image

> Clone
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,509 @@
...
+              <li class="cate31"><a href="#">Chrome will soon block ads that unnecessarily tax your CPU and battery</a></li>
+              <li class="cate31"><a href="#">Chrome will soon block ads that unnecessarily tax your CPU and battery</a></li>
+              <li class="cate31"><a href="#">Chrome will soon block ads that unnecessarily tax your CPU and battery</a></li>
+              <li class="cate31"><a href="#">Chrome will soon block ads that unnecessarily tax your CPU and battery</a></li>
+            </ul>
+            </div>
+          </div>
+
+          <div class="latest_deals1">
+            <p class="latest_deals11">Latest Deals ></p>
```
Comment (lucasmazo32) [inline index.html:419]: - [ ] Kindly fix the layout for latest deals for medium screens.

> Source
image

> Clone
image

**Review by lucasmazo32 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

You are doing a great job so far, and the project is looking good. 

For the project to get approved, kindly implement the changes mentioned below.

### Happy coding! :computer:

## Lucas Mazo

**Comment by SigmaSam** [comment]
New Features.
- Improved responsiveness for small and big screens.
- Add medium screen breakpoint and design.
- Fixed issues mentined in previus reviews.

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi **TEAM**, congratulations on fixing all the changes required by the previous TSE's :+1:  Your project now meets the minimum requirements to get approved :slightly_smiling_face: I, therefore, approve your project. You can go ahead and merge to master.
Do not forget to submit the project completion form

_Happy coding_ :man_technologist: 
Good luck with your next projects
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**